### PR TITLE
[ONNX] add test images to repo (#61553)

### PR DESCRIPTION
**Pull request by BowenBao**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #63249 Enhance shape (#60335)
* #62765 [ONNX] Suppport torch.dot and torch.nn.utils.spectral_norm (#62596)
* #62764 [ONNX] Update repeat_interleave for dynamic repeats (#59979)
* #62763 [ONNX] Fix an issue that optimizations might adjust graph inputs unexpectedly. (#61280)
* #62762 [ONNX] Fix controlflow shape inference with contrib op (#60707)
* **#62761 [ONNX] add test images to repo (#61553)**
* #62760 [ONNX] Fix for batchnorm training op mode (#52758)
* #62759 [ONNX] Remove aten parameter (#61652)
* #62758 [ONNX] Add support for opset14 in PT-ONNX exporter (#59486)
* #62757 [ONNX] Support lstm_cell symbolic (#61476)

This is better than the status quo:
* Test doesn't download files from the internet -> faster and more
  reliable.
* Test doesn't leave the git working directory dirty.

Rather than using the original images, I've copied some images from
the pytorch/vision repo. This will keep the tests in the two repos
in sync, while avoiding adding new assets to the vision repo.

See https://github.com/pytorch/vision/pull/4176.

### CODE REVIEW

**Comment by garymm** [comment]
@bowenbao this can be dropped / reverted. It got merged to master via #63717
